src/Config/StageConfigInterface.php

```diff
@@ -36,6 +36,14 @@
     const VAR_SKIP_HTML_MINIFICATION = 'SKIP_HTML_MINIFICATION';
     const VAR_SCD_MATRIX = 'SCD_MATRIX';
 
+    /**
+     * Environment variables.
+     */
```

Comment:
Could you please move this new varaibles under new config section like:

```
stage: []
system: 
-
-
```

This will make those variables very isolated from stage variables